resources/providers/azurelib/inventory/provider.go

```diff
@@ -37,14 +40,15 @@ import (
 type Provider struct {
 	log           *logp.Logger
 	client        *AzureClientWrapper
-	subscriptions map[string]string
+	subscriptions map[string]string // Key: subscription ID, value: display name
```

Comment:
```suggestion
	subscriptions map[string]string
```

This is going to be refactored in https://github.com/elastic/cloudbeat/pull/1559 anyway